test/e2e/tests/get-protocolp-parameters.e2e.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, test } from 'vitest';
```

Comment:
typo in filename? get-protocol**p**-parameters.e2e.test.ts

Reply:
c9ce170 should solve this